templates/miq-template.yaml

```diff
@@ -176,6 +241,10 @@ objects:
            name: "${NAME}-region"
            persistentVolumeClaim:
              claimName: ${NAME}-region
+         -
+           name: "${NAME}-assets"
```

Comment:
Then we need all of the application and its dependencies on this container... :cry: